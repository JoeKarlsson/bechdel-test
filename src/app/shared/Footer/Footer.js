import React from 'react';
import styles from './Footer.scss';

const Footer = () => (
	<footer className={styles.footer}>
		<div className={styles.footer_content}>
			<a
				target="_blank"
				rel="noopener noreferrer"
				href="https://github.com/JoeKarlsson/bechdel-test"
			>
				<p>a karlsson production.</p>
			</a>
		</div>
	</footer>
);

export default Footer;
