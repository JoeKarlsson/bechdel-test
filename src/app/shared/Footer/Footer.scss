$white: white;
$red: tomato;

.footer {
	background-color: $red;

	.footer_content {
		padding: 30px;
		text-align: center;
	}

	a {
		color: $white;
	}
}
