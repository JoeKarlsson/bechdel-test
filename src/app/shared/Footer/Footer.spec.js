import React from 'react';
import Footer from './Footer';

describe('Footer Page', () => {
	describe('rendering', () => {
		it('match the snapshot', () => {
			const wrapper = shallow(<Footer />);
			expect(wrapper).toMatchSnapshot();
		});
	});
});
