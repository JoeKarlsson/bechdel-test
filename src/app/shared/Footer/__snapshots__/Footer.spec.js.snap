// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer rendering initial state is rendered 1`] = `
<footer
  className="footer"
>
  <div
    className="footer_content"
  >
    <a
      href="https://github.com/JoeKarlsson/bechdel-test"
      rel="noopener noreferrer"
      target="_blank"
    >
      <p>
        2018
         - a karlsson production.
      </p>
    </a>
  </div>
</footer>
`;
